example/android/app/build.gradle

```diff
@@ -41,7 +41,7 @@ android {
     defaultConfig {
         applicationId "io.ably.flutter.example"
         minSdkVersion 19
-        targetSdkVersion 30
+        targetSdkVersion 32
```

Comment:
Is there a particular reason why you've chosen `32` and not `31`? :thinking: I think that version `32` isn't released yet or is currently in beta, right? Here's a list of the versions https://developer.android.com/studio/releases/platforms :wink:

Reply:
Android Studio suggest version 32, when it's set to 31. I'm not using any Preview/Canary version, so I thought that the suggested one is the latest stable 🤔 I'll downgrade to 31 then 🙂

![Zrzut ekranu 2022-02-4 o 14 11 58](https://user-images.githubusercontent.com/9201746/152534843-868f042e-b572-4b03-91de-6e016ccf6e1a.png)

Reply:
That's strange :sweat_smile: I've checked and in the AAT project Android Studio suggests me version 31. I wonder why :thinking: 
![image](https://user-images.githubusercontent.com/62378170/152535447-55b8193a-687a-4cc7-a2e6-af8ac1f909f3.png)

Reply:
@KacperKluka downgraded to SDK 31 in https://github.com/ably/ably-flutter/pull/312/commits/c198df56bf5e142d9592284818fd53e5e895fa9c